docs/semgrep-secrets/view-triage.md

```diff
@@ -59,27 +59,23 @@ You can triage findings in bulk by performing the following steps:
 
 1. Begin by ensuring that you display all **Open** findings.
 2. Apply filters with as much specificity as possible. You may have to perform bulk triage several times. By starting with the most specific cases, and closing the findings from those specific cases, you are able to narrow down findings as you work from specific to broad filter criteria.
-3. Click the bulk select check ox.
-4. Click **Triage**, then your selected triage state, typically **Ignore**.
+3. Click the bulk select check box.
+4. Click **Triage**, then your selected triage state, such as **Reviewing** or **Ignored**.
 5. Optional: Repeat this procedure to triage all open findings.
 
 
-## Receive findings in through PR and MR comments
+## Receive findings through PR and MR comments
 
 In addition to viewing your results in Semgrep AppSec Platform, you can set up PR or MR comments from Semgrep, which allows you to view findings-related information directly in your pull requests and merge requests.
 
 To receive PR or MR comments, ensure that:
 
 * You have set up [comments](/category/pr-or-mr-comments) as part of your core deployment.
-* You have defined which rules should be in Allow, Comment, or Block mode in the [Policies](/semgrep-secrets/policies) page.
+* You have defined which rules and validation states should be in Allow, Comment, or Block mode in the [Policies](/semgrep-secrets/policies) page.
 
 ![Semgrep Secrets finding in a PR comment](/img/secrets-pr-comment.png)
 **_Figure._** Semgrep Secrets finding in a PR comment.
 
 :::info
-Define which rules should be in allow, comment, or block mode in the [Policies](/semgrep-secrets/policies) page.
+Define which rules and validation states should be in Allow, Comment, or Block mode in the [Policies](/semgrep-secrets/policies) page.
```

Comment:
I'm actually not sure we need this line, given there's a similar one on 74. Let me know if deleting it would be better.

Reply:
We can remove it!